cmd/dptp-controller-manager/main.go

```diff
@@ -25,37 +25,27 @@ import (
 	controllerruntime "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
 	ctrlruntimelog "sigs.k8s.io/controller-runtime/pkg/log"
-	"sigs.k8s.io/controller-runtime/pkg/manager"
-	"sigs.k8s.io/yaml"
 
 	imagev1 "github.com/openshift/api/image/v1"
 
-	"github.com/openshift/ci-tools/pkg/api/secretbootstrap"
-	"github.com/openshift/ci-tools/pkg/controller/imagepusher"
 	"github.com/openshift/ci-tools/pkg/controller/promotionreconciler"
-	"github.com/openshift/ci-tools/pkg/controller/registrysyncer"
 	"github.com/openshift/ci-tools/pkg/controller/secretsyncer"
-	secretsyncerconfig "github.com/openshift/ci-tools/pkg/controller/secretsyncer/config"
```

Comment:
remove pkg/controller/secretsyncer if you remove the plumbing

Reply:
Updated.